apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -172,6 +173,30 @@ static void vfs_rmdir(const char *dirpath);
 extern int mount_show(void);
 #endif
 
+#ifdef CONFIG_AUTOMOUNT_USERFS
+static void move_to_next_part(const char **par)
+{
+	/* Move to next part information. */
+	while (*(*par)++ != ',');
+}
+
+static int fs_get_mount_partno(void)
+{
+	int partno = 0;
+
+	char *partname = CONFIG_FLASH_PART_NAME;
```

Comment:
In my opinion, we'd better avoid access internal config where in lower level.
Like a pipe or something related to syscall, that is ok.
But this case had better hided to user lvl, because it using by mtd level which never open to user.
Does previous version has some problems?